src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/jobservices/FindProblemMessageJobTest.java

```diff
@@ -29,8 +30,11 @@ public class FindProblemMessageJobTest {
 
     private FindProblemMessageJob findProblemMessageJob;
 
+    @Mock
```

Comment:
Please drop Mock annotation or mock readTree method and remove the instance creation.